tool/tctl/common/tctl.go

```diff
@@ -294,6 +294,17 @@ func applyConfig(ccf *GlobalCLIFlags, cfg *service.Config) (*AuthServiceClientCo
 			return nil, trace.Wrap(err)
 		}
 	}
+	if err = config.ApplyFileConfig(fileConf, cfg); err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	// --auth-server flag(-s)
+	if len(ccf.AuthServerAddr) != 0 {
+		cfg.AuthServers, err = utils.ParseAddrs(ccf.AuthServerAddr)
```

Comment:
Missing a patch here?

Reply:
Oof. Yeah, cherry picking had such a mess of conflicts that I just re-did this code by hand. I'll add the patch for not overwriting cfg.AuthServers on err.

Reply:
Will do in the future. Thanks for the catch @codingllama!